GitHub-resources/DIME-GitHub-Guides/clone-location.md

```diff
@@ -0,0 +1,211 @@
+Remember to select the clone folder in this step,
+and not just the content of the clone.
+In the example we have used,
+this means that you should select the folder called `lyrics-clone`.
+See the second image below.
+
+<img src="https://github.com/worldbank/dime-github-trainings/blob/move-clone/GitHub-resources/DIME-GitHub-Guides/img/clone-not-found.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+<img src="https://github.com/worldbank/dime-github-trainings/blob/move-clone/GitHub-resources/DIME-GitHub-Guides/img/clone-new-location.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+If you do not see the screen where you are asked to locate the clone,
+then go to the menu of GitHub Desktop and select "_File_" -> "_Add local repository_".
+Then use the menu that comes up and select the clone folder in the new location.
+GitHub desktop will then identify this is the new location of the same clone,
+and it will detect that this is a clone of a repository hosted on GitHub.com.
+
+## Why are synced folders a bad location for my clone?
+
+In short, syncing software and git are both collaboration tools,
+but they work very differently.
+And neither of the tools are implemented to work together,
+which can cause many different types of weird errors.
+
+#### Shared synced folders
+
+Putting a clone in a shared synced folder quickly leads to issues if
+more than one person is expected to work on the same clone in that shared folder.
+For example, if one user changes branch in the middle of someone else running the code,
+then all weird errors will happen as the files are changed by git.
+It is probably easy to see how this leads to errors
+and there is no standard workflow where this should ever be the norm.
+
+#### Non-shared synced folders
+
+What about non-shared synced folders? Then there will never be an issue with another user.
+While that is true, there are still errors than can happen in any kind of synced folder.
+And while these errors are more rare they are
+likely to cause major disruptions when they happen.
+
+When syncing software detects a file with edits to be synced to the cloud,
+it blocks access to that file for a fraction of a second.
+That is fine when it is only much slower humans that also competes for access to those files.
+However, git is not a human and in certain git actions,
+git can make many updates in a fraction of a second
+to the technical files in the `.git` folder.
+There is always a `.git` folder in all clone folders,
+but depending on the settings of your computer,
+this `.git` folder might or might not be visible to you.
+
+As you use different features of GitHub Desktop,
+series of git commands are executed in the background in rapid succession.
+This means that git might do an edit to a file,
+then do something else and then need to make a second edit to that first file.
+This can happen within the same fraction of a second under which  
```

Comment:
```suggestion
This can happen within the same fraction of a second under which
```
Deleting two blank spaces, in markdown they are interpreted as a line break